README.md

```diff
@@ -1,2 +1,34 @@
 # actions-project-version-check
-GitHub Action that checks if the project version has been updated
+
+This action checks if the project version has been updated in your pull request. The action will request the file content (from file with name from envrionment variable `file-to-check`) from the pull request target branch and parse the project version. After that the local project version will be checked against the targets one with [semver-diff](https://www.npmjs.com/package/semver-diff). If the result is `undefined`, the action fail.
```

Comment:
Im zweiten Satz liest sich das irgendwie komisch mit den "vielen" `from`s. Evtl. kann man das ja noch ein bisschen optimieren

..the action fail[s].